content/help/ops/security/repairing-citadel/index.md

```diff
@@ -5,6 +5,15 @@ weight: 10
 keywords: [security,citadel,ops]
 ---
 
+`Currently we only support single Citadel instance. Running multiple Citadel instances may introduce
+race condition and cause system outage`.
+
+Citadel is not on the critical path of data plane. The default workload certificate lifetime is 3
```

Comment:
Citadel is not a critical data plane component.

Reply:
done.